functions/supplier/rabbit-supplier/src/main/java/org/springframework/cloud/fn/supplier/rabbit/RabbitSupplierConfiguration.java

```diff
@@ -77,9 +79,6 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source,
 	@Autowired
 	private RabbitProperties rabbitProperties;
 
-	@Autowired
-	private ObjectProvider<ConnectionNameStrategy> connectionNameStrategy;
```

Comment:
This was only in here to appease the signature of the connection factory creator. The code immediately turned arounds and overrides the naming strategy so I removed this.